src/App.java

```diff
@@ -9,6 +9,7 @@ public static void main(String[] args) {
         names.add("James");
         names.add("Kiel");
         names.add("Irina");
+        names.add("Bob");
```

Comment:
@smashdevcode Ticket #45 said to add Fred's name, not Bob's. Could you please correct and resubmit

Reply:
Oh no... working on this change now...